### ISSUE TITLE
Problems in Firefox for Linux (LMDE 4)

### ISSUE THREAD

**Issue by liberii** [FilipePS/Traduzir-paginas-web#158]
1) I have to click three times on the icon before the window appears.
2) Translation of selected text does not work.
3) Shortcuts do not work and cannot be changed.

**Comment by FilipePS** [comment]
1. Probably something related to your computer's performance. Did this only happen after the last update of the extension? Clear the translation cache and tell me if the problem resolves.
2. You leave the translate selected text button hidden right? If so, this bug has already been fixed.
3. About not being able to change the shortcut key. This happens with those who have a keyboard with a Russian, Chinese, Japanese layout ... This bug has already been fixed.
Now on the shortcut does not work. It is also related to the keyboard layout. Basically in some keyboard layouts, no browser extension can use "Ctrl + Alt` as a shortcut. So if this shortcut doesn't work on your keyboard, change it.

**Comment by liberii** [comment]
1. It works
2. It works
3. It works. It was difficult to find how to change the shortcut. You need to explain somewhere how to change shortcuts. Congratulations for your work. This is the best extension for translating web pages that I have used on Firefox.

**Comment by FilipePS** [comment]
On the configuration page itself, you have options to change the shortcut keys. I don't understand what can be difficult about this.

**Comment by liberii** [comment]
When I click in the "Enter a shortcut" field, nothing happens. I was expecting the writing to disappear and that vertical bar to appear so I could enter the shortcut, but the writing doesn't disappear and the bar doesn't appear. I tried anyway and found that I can enter the shortcut by holding down at the same time ctrl and a number. So I got working shortcuts.

![1](https://user-images.githubusercontent.com/71891088/118190323-79d07080-b432-11eb-9713-04a710d9ee5e.png)
